datadog_checks_downloader/datadog_checks/downloader/download.py

```diff
@@ -282,11 +319,14 @@ def __get_versions(self, standard_distribution_name):
         # version: {python_tag: href}
         wheels = collections.defaultdict(dict)
 
-        try:
-            # NOTE: We do not perform in-toto inspection for simple indices; only for wheels.
-            index_abspath, _ = self.__download_with_tuf(index_relpath)
-        except TargetNotFoundError:
-            raise NoSuchDatadogPackage(standard_distribution_name)
+        if self.__disable_verification:
+            index_abspath = self.__download_without_tuf(index_relpath)
+        else:
+            try:
+                # NOTE: We do not perform in-toto inspection for simple indices; only for wheels.
+                index_abspath, _ = self.__download_with_tuf(index_relpath)
+            except TargetNotFoundError:
+                raise NoSuchDatadogPackage(standard_distribution_name)
```

Comment:
This is out of scope for the current PR, but shouldn't this error be more indicative that (1) it's for an integrations, not package and (2) there's no tuf target (vs the integration doesn't exist at all)?

Reply:
My guess is that right now if you make a typo in the package name, this is where the downloader will fail. So it makes sense to say "no such Datadog package". However it could be a nice thing indeed when no TUF target is found for this integration to do a quick look for the S3 bucket directory, to try to tell if it's a typo in the integration name or a bug in the TUF metadata.